### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by cheshirecatalyst** [jamesmacfie/space-tab#4]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.activate (/Users/emris/.atom/packages/space-tab/lib/space-tab.coffee:3:7)
```

**Comment by cheshirecatalyst** [comment]
See also:
Atom.Object.defineProperty.get - /Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11
Object.activate - /Users/emris/.atom/packages/space-tab/lib/space-tab.coffee:4:7

**Comment by jamesmacfie** [comment]
Fixed in v0.3.0. Thanks
